📝 Update `en/docs/tutorial/debugging.md`

nilslindemann opened the pull request:
This chapter was outdated:

* VS Code now has a special debug configuration for FastAPI, so calling `uvicorn` from the `myapp.py` is not necessary.
* The UI of PyCharm has changed, and it is also easier to start the debugger.

I kept the `__name__ == "__main__"` part, but rewrote the rest.

Feel free to modify as you want.

#10591 is a translation of this.

Commit:
Update debugging.md

Commit:
Update code examples

Commit:
Update images

tiangolo commented:
📝 Docs preview for commit 795445b2357c86a6b2adf684b698af92d05b5d4c at: https://8f6d59a7.fastapitiangolo.pages.dev

Commit:
"file" -> "module"

tiangolo commented:
📝 Docs preview for commit a0c83b2ee36430d018b622aceb9d5e6b748af420 at: https://4c10a07d.fastapitiangolo.pages.dev

Commit:
Be more in sync with original `__name__=="__main__"` chapter

When originally rewriting the text I modified the German translation of the original English text and, when done, I translated everything back to English. Some things got lost while doing that, which this commit fixes.

Also undo the changes of the previous commit.

tiangolo commented:
📝 Docs preview for commit 04c8adff3ffaa4d7186a404cfe8c3dc986459ebd at: https://65312a71.fastapitiangolo.pages.dev

Commit:
Better wording

tiangolo commented:
📝 Docs preview for commit 6442719ec65177c1d8b96b53182893386323334d at: https://d15b381d.fastapitiangolo.pages.dev

Commit:
Better wording

tiangolo commented:
📝 Docs preview for commit 8f389692462cb21e77d8c2836505a3a172949fda at: https://e9051d67.fastapitiangolo.pages.dev

Commit:
And another rewording

tiangolo commented:
📝 Docs preview for commit a0676bb22a3084ef939fc615058d196a173fd2d7 at: https://305990b4.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into update-en/docs/tutorial/debugging.md

tiangolo commented:
📝 Docs preview for commit b039a661051b2e943ed7799092743029123c2ff2 at: https://e83d55cb.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into update-en/docs/tutorial/debugging.md

tiangolo commented:
📝 Docs preview for commit f74ed2c9184d7ff50885c52d31d6cf908b6bb2e2 at: https://f9ddd104.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into update-en/docs/tutorial/debugging.md

tiangolo commented:
📝 Docs preview for commit 46ebcbe40e98828a4eb24ade6517f4ef8cf1584c at: https://7c08282b.fastapitiangolo.pages.dev

alejsdev committed:
✏️ Update debugging.md

pre-commit-ci[bot] committed:
🎨 [pre-commit.ci] Auto format from pre-commit.com hooks

Commit:
Merge branch 'tiangolo:master' into update-en/docs/tutorial/debugging.md

nilslindemann commented:
Thanks for the review, @alejsdev

tiangolo commented:
📝 Docs preview for commit 8ccde00a628d66e096ac4ea25bfdeb79e43917d1 at: https://d9f7b8ec.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into update-en/docs/tutorial/debugging.md

tiangolo commented:
📝 Docs preview for commit f641f31b3cf0caae9e0697eb55de08b00dd13a47 at: https://9a4f6523.fastapitiangolo.pages.dev

YuriiMotov committed:
Merge branch 'master' into update-en/docs/tutorial/debugging.md

YuriiMotov reviewed (changes requested):
@nilslindemann, thanks for your efforts!

I think there are a couple of things that need to be improved. 🧐
Would you like to continue working on this?

nilslindemann commented on docs/en/docs/tutorial/debugging.md:
```diff
@@ -1,16 +1,93 @@
+
+* Next, choose "FastAPI" from the options.
+
+* In the subsequent dropdown, enter the name of your application file, here `myapp.py` (This is only necessary because VS Code did not find a `main.py` file).
```
Thanks, Yuri, I currently don't have time for this. Very probable that this is meanwhile outdated again. After all, the PR is nearly two years old. I will close this issue for now, and may create a new one when there is time.

YuriiMotov commented:
@nilslindemann, thank you for your contribution!
Sorry for not being able to review it in time..
